src/log-error.js

```diff
@@ -19,9 +19,6 @@ function getErrorObject(value) {
 
 export function logError(error) {
     if (window.onerror) {
-        // set timeout since jasmine doesn't expect window.onerror to be called from its own context
-        setTimeout(() => {
-            window.onerror(getErrorObject(error));
-        });
+        window.onerror(getErrorObject(error));

```

Comment:
add a try-catch in case window.onerror throws ?

Reply:
IMFO we should not. If the global onerror handler fails we should let it fail, as this means something is broken on the app side, not the lib.